sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/AggregateBenchmark.scala

```diff
@@ -34,621 +34,539 @@ import org.apache.spark.unsafe.map.BytesToBytesMap
+        spark.range(N).groupBy().agg("id" -> "stddev").collect()
+      }
 
-    runBenchmark("kurtosis", N) {
-      sparkSession.range(N).groupBy().agg("id" -> "kurtosis").collect()
+      runBenchmarkWithCodegen("kurtosis", N) {
+        spark.range(N).groupBy().agg("id" -> "kurtosis").collect()
+      }
     }
 
-    /*
-    Using ImperativeAggregate (as implemented in Spark 1.6):
-
-      Intel(R) Core(TM) i7-4558U CPU @ 2.80GHz
-      stddev:                            Avg Time(ms)    Avg Rate(M/s)  Relative Rate
-      -------------------------------------------------------------------------------
-      stddev w/o codegen                      2019.04            10.39         1.00 X
-      stddev w codegen                        2097.29            10.00         0.96 X
-      kurtosis w/o codegen                    2108.99             9.94         0.96 X
-      kurtosis w codegen                      2090.69            10.03         0.97 X
-
-      Using DeclarativeAggregate:
-
-      Intel(R) Core(TM) i7-4558U CPU @ 2.80GHz
-      stddev:                             Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
-      -------------------------------------------------------------------------------------------
-      stddev codegen=false                     5630 / 5776         18.0          55.6       1.0X
-      stddev codegen=true                      1259 / 1314         83.0          12.0       4.5X
-
-      Intel(R) Core(TM) i7-4558U CPU @ 2.80GHz
-      kurtosis:                           Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
-      -------------------------------------------------------------------------------------------
-      kurtosis codegen=false                 14847 / 15084          7.0         142.9       1.0X
-      kurtosis codegen=true                    1652 / 2124         63.0          15.9       9.0X
-    */
-  }
-
-  ignore("aggregate with linear keys") {
-    val N = 20 << 22
+    runBenchmark("aggregate with linear keys") {
+      val N = 20 << 22
 
-    val benchmark = new Benchmark("Aggregate w keys", N)
-    def f(): Unit = {
-      sparkSession.range(N).selectExpr("(id & 65535) as k").groupBy("k").sum().collect()
-    }
+      val benchmark = new Benchmark("Aggregate w keys", N, output = output)
 
-    benchmark.addCase(s"codegen = F", numIters = 2) { iter =>
-      sparkSession.conf.set("spark.sql.codegen.wholeStage", "false")
-      f()
-    }
+      def f(): Unit = {
+        spark.range(N).selectExpr("(id & 65535) as k").groupBy("k").sum().collect()
+      }
 
-    benchmark.addCase(s"codegen = T hashmap = F", numIters = 3) { iter =>
-      sparkSession.conf.set("spark.sql.codegen.wholeStage", "true")
-      sparkSession.conf.set("spark.sql.codegen.aggregate.map.twolevel.enabled", "false")
-      sparkSession.conf.set("spark.sql.codegen.aggregate.map.vectorized.enable", "false")
-      f()
-    }
+      benchmark.addCase(s"codegen = F", numIters = 2) { _ =>
```

Comment:
`s"codegen = F"` -> `"codegen = F"`?

Reply:
Thanks @dongjoon-hyun I plan add `EmptyInterpolatedStringChecker` to scalastyle-config.xml to avoid this issue: [SPARK-25553](https://issues.apache.org/jira/browse/SPARK-25553)